### ISSUE TITLE
Cannot Patch Binary / assemble instructions with radare2 

### ISSUE THREAD

**Issue by shard77** [radareorg/radare2#21631]
## Environment
Arch Linux
Radare2 Version 5.8.2 - Using https://archlinux.org/packages/community/x86_64/radare2/

```sh
r2 -w <binary>
aaa
V
A
mov rdx, rsp 
ENTER
ERROR: cannot assemble 'mov rdx, rsp' <at line <line> >
```

## Description

Cannot assemble any instructions with `A` or the `pa` command.
The problem doesn't seem to come from the version from what I've seen.
I don't get any assembled output, and radare2 throws the error even with instructions that have no symbols.

![image](https://user-images.githubusercontent.com/106669955/233778300-ee31805c-872e-4dd4-a97d-f72a9b355069.png)

**Comment by trufae** [comment]
Cant reproduce, but from the error it looks like its trying to use gas instead of the default assembler shipped in r2. Looks more like a packaging issue from arch rather than a bug in r2. If you install it from or from last release the problem will be gone I guess.

let me check how arch is building r2..

**Comment by trufae** [comment]
I can see several issues in the package from arch.. so better install it from sources or report that back to them.

- libuv is no longer a dependency
- Capstone4 is 6 year old dependency and misses lots of important instructions mainly on x64 and arm64 so the analysis will be wrong in most modern binaries (spectre protections and pac stuff for example)
- 5.8.2 is old, last release is 5.8.4 and it have several bugs fixed (ill release 5.8.6 in a week)
- Its creating an empty config.mk which is also wrong and probably the reason why its missing plugins
- libmagic from gnu is like 10 times slower than the implementation shipped in r2 which is based on openbsd code.

**Comment by shard77** [comment]
Hey! Thanks for your quick response.
I'll try to contact the package owner, for now I'll install it from git.
Thanks!
